### ISSUE TITLE
Documentation refers to vulnerabilityName which isn't available

### ISSUE THREAD

**Issue by vandmo** [dependency-check/DependencyCheck#3948]
**Describe the bug**
Documentation not up to date.

**Log file**
```
[ERROR]         InitializationException: Warn initializing the suppression analyzer: Failed to load src/main/config/suppressions.xml, caused by org.owasp.dependencycheck.xml.suppression.SuppressionParseException: org.xml.sax.SAXException: Line=7, Column=37: cvc-complex-type.2.4.a: Invalid content was found starting with element 'vulnerabilityName'. One of '{"https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.1.xsd":cpe, "https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.1.xsd":cve, "https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.1.xsd":cwe, "https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.1.xsd":cvssBelow}' is expected.. 
```

**Version of dependency-check used**
6.5.2

**Expected behavior**
https://jeremylong.github.io/DependencyCheck/general/suppression.html shouldn't reference `<vulnerabilityName>` since that isn't supported.

**Comment by jeremylong** [comment]
Just double checked and per the schema and source code `vulnerabilityName` is allowed if referencing the 1.3 schema: https://github.com/jeremylong/DependencyCheck/blob/40630a16532e57c8315b3f155010c9628ccca078/core/src/main/resources/schema/dependency-suppression.1.3.xsd#L50

Does the top of your suppression file contain a reference to the 1.1 or 1.2 schema?

**Comment by aikebah** [comment]
Their suppressionfile indeed still uses the 1.1 version, which is obvious from the error message regarding the namespace of the expected elements `One of '{"https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.1.xsd":cpe, `

**Comment by jeremylong** [comment]
@vandmo simple fix - update the schema version to 1.3 and everything should work.

**Comment by vandmo** [comment]
I completely missed that, thanks! :bow:
